wordpress-mobile/gutenberg-mobile
# Upgrade React Native `0.71.11` - Android changes

**Related PRs:**
* https://github.com/WordPress/gutenberg/pull/51289
* https://github.com/wordpress-mobile/WordPress-Android/pull/18613

**To test:**
* Observe that all CI jobs pass, except the ones related to iOS. The failures on iOS will be solved in https://github.com/wordpress-mobile/gutenberg-mobile/pull/5881.
* Optionally, follow testing instructions from https://github.com/WordPress/gutenberg/pull/51303.

PR submission checklist:
